webapp/components/signup/components/signup_email.jsx

```diff
@@ -40,7 +40,7 @@ export default class SignupEmail extends React.Component {
         this.state = this.getInviteInfo();
     }
 
-    componentDidMount() {
+    static componentDidMount() {
```

Comment:
@grundleborg fixed in https://github.com/mattermost/platform/pull/7025/commits/55035b27a285b72668323a107bb99b0939da0afd